packages/react-core/src/components/Alert/AlertIcon.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/Alert/styles.css';
+import CheckCircle from '@patternfly/react-icons/dist/esm/icons/check-circle';
+import ExclamationCircle from '@patternfly/react-icons/dist/esm/icons/exclamation-circle';
+import ExclamationTriangle from '@patternfly/react-icons/dist/esm/icons/exclamation-triangle';
+import InfoCircle from '@patternfly/react-icons/dist/esm/icons/info-circle';
```

Comment:
These need to be handled by the babel plugin.  so would would write them like you had it with:
```
import { .CheckCircle, ExclamationCircle } from '@patternfly/react-icons';
```
The reason we cannot import the esm files directly, which would be much better, is that commonjs environments would not be able to load them.  It would require babel or webpack bundling for server rendering or something similar.  the babel plugin will take care of rewriting the imports to be from `js` (commonjs) or `esm`(es modules).  Once node v10 goes LTS and ES Modules are more widely supported we can probably drop the commonjs all together.